build/compose.yml

```diff
@@ -30,9 +30,8 @@ services:
     build: 
       context: ..
       dockerfile: build/node.dev.Dockerfile
-    env_file:
-      - path: ../dev.env
-        required: false
+    env_file: ../dev.env
     depends_on:
       - habitat_frontend
-
+    extra_hosts:
```

Comment:
ooh what is this for? can you leave a comment

Reply:
This is just to get communication between containers working for Linux. On MacOS the docker internal host already pointed to the host machine by default, so it wasn't necessary before. I just got development in Linux working on this PR.